beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,279 @@
+
+
+<hr>
+
+## Manual Azure Portal Deployment Steps
+
+<details>
+<summary> Manual Azure Portal Deployment Steps</summary>
+
+### Step 1: Configure the `scimsession` credentials for passing to Container App
+
+The `scimsession` credentials will be saved as an secret variable in Container App. These credentials have to be Base64-encoded to pass them into the environment, but they're saved as a plain-text file when you download them or save in 1Password during the setup.
+
+> **Note**
+    >
+    >You will need to update the path to the scimsession file you download in the getting started section.
+
+1. Get the Base64 encoded contents of your `scimsession` file, _(using the bash or PowerShell syntax for the commands)_: 
+   
+   - Using bash (Cloud Shell, macOS, or Linux):
+     ```bash
+     cat ./scimsession | base64
+     ```
+   - Using PowerShell (Cloud Shell, Windows, macOS, or Linux):
+     ```pwsh
+     [Convert]::ToBase64String([IO.File]::ReadAllBytes((Join-Path $PWD.Path 'scimsession')))
+     ```
+   Copy the output value from the terminal to your clipboard. It will be needed to create the secret for the deployment.
+2. Save the encoded value in your 1Password account for future use: 
+   - Locate the "scimsession file" item in your 1Password account.
+   - Click Edit.
+   - Select "Add another field".
+   - Select Password.
+   - Rename the Password label above the value you just pasted, to something like `Base64 value`. 
+   - Paste the Base64 value from your clipboard into the password field. 
+   - Select **Save**.
+
+### Step 2: Deploy your base SCIM bridge within Azure Container Apps
+
+1. Create a new Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps).
+2. Either use an existing Resource Group or select Create New under the **Resource Group** section.
+3. Enter the **Container app name**, for example *op-scim-con-app*.
+4. Select your **Region** for the deployment.
+5. Either use the existing Container App Environment generated or select Create New under the **Container Apps Environment** Section.
+    1. If creating a new Container App Environment, use a name that makes sense for your organization, for example op-scim-con-app-env.
+    2. Leave **Consumption** selected, and **Zone Redundancy** disabled.
+    3. Select **Create** on the create container apps environment window.
+6. Select **Next : Container >** to continue the container app creation.
+    1. Uncheck **Use quickstart image**.
+    2. Change the name of the container to *op-scim-bridge*.
+    3. Select **Docker Hub** for the image source.
+    4. Enter **1password/scim:v2.8.2** for the **Image and Tag**.
+    5. Leave the CPU and Memory as 0.25 CPU cores, 0.5 Gi memory.
+    6. Add the following Environment variables:
+        1. **OP_SESSION** currently with of **""** for now.
+        2. **OP_REDIS_URL** with a value of **redis://localhost:6379**
```

Comment:
Ditto as above, things that should be entered as is use backticks

Reply:
Applying formatting suggestion

```suggestion
        1. `OP_SESSION` currently with of `""` for now.
        2. `OP_REDIS_URL` with a value of `redis://localhost:6379`
```